BaseFilter/sections/css_extended.txt

```diff
@@ -6,6 +6,8 @@
 ! Bad: example.org##main > div.ad (should be in specific.txt)
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/133157
+work.ink#?#.content .wk-container:has(.container-content:not(:has(> *)))
```

Comment:
It does not block leftovers on my end.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/198968956-c42e8017-3b91-43df-a505-2ed011813c30.png)

</details><br/>

Try this `work.ink#?#.wk-container:has(> div.container-content > div[id^="waldo-tag-"]:first-child)`

Reply:
So strange. I see the website as the following screenshot:

![Screensh1](https://user-images.githubusercontent.com/98787049/198973339-63659723-0a3b-4b4f-8761-31d8d4d3f2fa.png)
![Screenshot from 2022-10-31 09-13-11](https://user-images.githubusercontent.com/98787049/198973499-d060594d-91d8-4698-aad2-0f9dce67f4b7.png)

Reply:
Yep, it is. Did the rule above help on your end?

Reply:
But, your extended css rule works on my end too.
I will use yours.